02_assignments/assignment_2.ipynb

```diff
@@ -265,8 +300,11 @@
         "\n",
         "def detect_problems(file_path):\n",
         "  #YOUR CODE HERE: use patient_summary() to get the means and check_zeros() to check for zeros in the means\n",
-        "\n",
-        "  return"
+        " means, has_zeros = patient_summary(file_path, 'mean')\n",
+        " if has_zero: \n",
+        "     return True\n",
+        " else:\n",
+        "     return False\n"
```

Comment:
You should run your code and see if it works before submitting it. Next time when you submit, remember to save your result in the file.